style.css

```diff
@@ -0,0 +1,683 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  max-width: 100%;
+}
+
+body {
+  max-width: 100%;
+  min-height: 100vh;
+  background-color: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+header {
+  background-color: #fff;
+}
+
+.amburger {
+  display: flex;
+  flex-direction: column;
+}
+
+.bar {
+  display: block;
+  width: 33px;
+  height: 3px;
+  margin: 2px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #162ae3;
+}
+
+.nav-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem;
+}
+
+section {
+  padding: 5rem 1.25rem 6rem;
+  background-color: #fff;
+  margin-bottom: 20%;
+}
+
+.major-container {
+  display: flex;
+  flex-direction: column;
+  border-bottom-left-radius: 80px;
+  background-image: url("images/footer/Header bg.png");
+  background-repeat: no-repeat;
+  background-position: inherit;
+  background-size: 100% 100%;
+  padding-top: 70px;
+  justify-content: center;
+}
+
+.section-head {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  width: 80%;
```

Comment:
- [ ] Please remove this `width` to make the section more similar to the original design. 👌😊